src/Fmt_ast.ml

```diff
@@ -3316,7 +3316,8 @@ and fmt_signature c ctx itms =
     @@ fun c -> fmt_signature_item c (sub_sig ~ctx i)
   in
   let fmt_grp itms = list itms "@\n" fmt_grp in
-  hvbox 0 (list grps "\n@;<1000 0>" fmt_grp)
+  let open Safe in
+  box ~box_singleton:true hvbox 0 (of_list grps double_linebreak ~f:fmt_grp)
```

Comment:
I'm not convinced by this example. With `~box_singleton:true`, this is exactly the same as before. This list should never be empty.

Reply:
Yes this example is not the best, I realized after modifying this part of the code that the box was needed even if there is only 1 element, but it is not the case for 99% of the other uses of boxes, I will write more examples.